Sources/Common/DataModel/Line/index.js

```diff
@@ -176,6 +187,7 @@ function vtkLine(publicAPI, model) {
       }
       if (outObj.t > 1.0) {
         outObj.t = 1.0;
+        outObj.betweenPoints = isBetweenPoints(outObj.t);
```

Comment:
I wouldn't recompute `betweenPoints` here

Reply:
Same as the previous comment.